src/components/FeatureViews/Scheduler/EditSessionView.js

```diff
@@ -235,11 +247,34 @@ function EditSessionView({course, session, editSelection}) {
                         />
                     </Grid>
 
+                    <Grid item xs={6}>
+                        <Typography variant="h5"> Duration </Typography>
+                        <Select
+                            onChange={handleDurationSelect}
+                            value={sessionFields.duration}
+                        >
+                            {courseDurationOptions.map((duration) => {
+                                return <MenuItem
+                                    value={duration}
```

Comment:
add a "key" attribute. Anytime you're using ".map()" and returning a component, do this. This is to improve performance. 

example:
```
courseDurationOptions.map((duration, index) => <MenuItem key={ index } value={ duration }> {duration </MenuItem>
```